Adds how to run tf-wrapper.sh script locally (#145)
terraform-google-modules/terraform-example-foundation: 2-environments/README.md

### Run terraform locally
1. Change into 2-environments folder.
1. Run `cp ../build/tf-wrapper.sh .`
1. Run `chmod 755 ./tf-wrapper.sh`
1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
1. Update backend.tf with your bucket from bootstrap. You can run
```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```.

> You can run in the 0-bootstap folder `terraform output gcs_bucket_tfstate` to obtain the bucket name.

Reviewer: could be 

> You can run `terraform output gcs_bucket_tfstate` in the 0-bootstap folder to obtain the bucket name.

> We will now deploy each of our environments(dev/prod/stage) using this script.

Reviewer: `stage` should be `nonprod`

Reviewer 2: correct, my apologies, I wrote it wrong when I suggested the change :)

> When checked into a VCS, each environment corresponds to a branch and only the corresponding environment is applied.

Reviewer: I think this part cloud be more direct saying something like: 
> when using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for `2-environments` step and only the corresponding environment is applied.

binding the branch name to the file system folder is done by the configuration on Cloud Build or Jenkins

    1. Run `terraform init`
    1. Run `terraform plan` and review output
    1. Run `terraform apply`

### Run tf-wrapper.sh script locally
1. Change into 2-environments folder.
1. Run `cp ../build/tf-wrapper.sh .`
1. Run `chmod 755 ./tf-wrapper.sh`
1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
1. Update backend.tf with your bucket from bootstrap. You can run
```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```

> 1. Run `./tf-wrapper.sh init dev`

Reviewer 2: Add a line or two before these explaining why we do `./tf-wrapper.sh apply branch`. Perhaps something like `We will now deploy each of our environments(dev/prod/stage) using this script. When checked into a VCS, each  environment corresponds to a branch and only the corresponding environment is applied.`

Author: Yeah! Adding your suggestion!

1. Run `./tf-wrapper.sh plan dev` and review output.
1. Run `./tf-wrapper.sh apply dev`
1. Run `./tf-wrapper.sh init nonprod`
1. Run `./tf-wrapper.sh plan nonprod` and review output.
1. Run `./tf-wrapper.sh apply nonprod`
1. Run `./tf-wrapper.sh init prod`
1. Run `./tf-wrapper.sh plan prod` and review output.
1. Run `./tf-wrapper.sh apply prod`

> If you got errors or made any changes on terraform code or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`

Reviewer 2: ```suggestion
If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
```

Author: Fixing the text!
